### ISSUE TITLE
07/11/23_YS_APK_ReverseGeocode_Text of the Disability carousel slider is incorrect_Parent Id - 2096

### ISSUE THREAD

**Issue by vinod96424** [nammayatri/nammayatri#4184]
#2096 

Steps to reproduce:

1. Launch the application
2. Tap on Disability banner on the home screen
3. Observe the Last carousel text.

Expected result:

Text of the Carousel slider should be as per figma

Actual Result:

Text of the Disability carousel slider is incorrect

Repro:5/5

![Screenshot_20231107_163710](https://github.com/nammayatri/nammayatri/assets/129082995/5bdf36e0-61e7-4549-97b2-0c6c78c35772)
<img width="330" alt="Screenshot 2023-11-07 at 5 03 56 PM" src="https://github.com/nammayatri/nammayatri/assets/129082995/bfbbf7f7-6298-4af1-b8de-5fb5220e6411">

**Comment by MercyQueen** [comment]
text needs to be updated in Figma.

**Comment by vinod96424** [comment]
Can you ask designers to update the same. Did they do the design review for the same?

**Comment by MercyQueen** [comment]
yes @vinod96424 I have asked Aromal to update the text in the Figma.
